src/com/projectkorra/projectkorra/firebending/combustion/Combustion.java

```diff
@@ -103,13 +103,15 @@ public static boolean removeAroundPoint(final Location loc, final double radius)
 	}
 
 	private void advanceLocation() {
-		ParticleEffect.FIREWORKS_SPARK.display(this.location, 5, Math.random() / 2, Math.random() / 2, Math.random() / 2, 0);
-		ParticleEffect.FLAME.display(this.location, 2, Math.random() / 2, Math.random() / 2, Math.random() / 2);
+		ParticleEffect.FIREWORKS_SPARK.display(this.location, 2, .001, .001, .001, 0);
+		ParticleEffect.CRIT.display(this.location, 3, Math.random() * 1.5, Math.random() * 1.5, Math.random() * 1.5, 0);

```

Comment:
Maybe throw in a few explosion particles every few iterations too

Reply:
I think it looks good the way it is

Reply:
I wanted it to be reminiscent of this, there arent any explosions until contact
![image](https://user-images.githubusercontent.com/19560788/86637508-82425a00-bf8a-11ea-8e3f-8c9ef8ba6a14.png)

Reply:
The explosion particle is basically that little ring you see around her beam, sweep attack might also look good there.

Reply:
The crit particle is supposed to be what represents that.

Reply:
Idk if I like that

Reply:
The explosion particle might be exactly what we are looking for, I would recommend we look into it.